Firestore/Example/Tests/Integration/API/FIRQueryTests.mm

```diff
@@ -1035,63 +1068,20 @@ - (void)testMultipleInOps {
   [self checkOnlineAndOfflineQuery:[[collRef queryWhereFilter:filter1] queryOrderedByField:@"a"]
                      matchesResult:@[ @"doc1", @"doc6", @"doc3" ]];
```

Comment:
You need to remove the OrderBy from this query, and change the expected result to `doc1, doc3, doc6`. This way the test can be easily re-enabled when prod is ready. See [this](https://cs.opensource.google/firebase-sdk/firebase-android-sdk/+/master:firebase-firestore/src/androidTest/java/com/google/firebase/firestore/QueryTest.java;l=1228-1232).